SnapshotTests/SnapshotTestCase.swift

```diff
@@ -50,15 +50,32 @@ extension SnapshotTestCase {
         }
     }
 
-    func nameForDevice(baseName: String? = nil) -> String {
+    func nameForDevice(_ baseName: NameForDevice = .portrait) -> String {
         let size = UIScreen.main.bounds.size
         let scale = UIScreen.main.scale
         let version = UIDevice.current.systemVersion
         let deviceName = "\(Int(size.width))x\(Int(size.height))-\(version)-\(Int(scale))x"
 
-        return [baseName, deviceName]
-            .compactMap { $0 }
-            .joined(separator: "-")
+        switch baseName {
+        case let .baseName(value):
+            return [value, deviceName]
+                .compactMap { $0 }
+                .joined(separator: "-")
+        case .portrait:
+            return "\(deviceName)"
+        case .landscape:
+            return "\(deviceName)-landscape"
+        }
+    }
+
+    enum NameForDevice: ExpressibleByStringLiteral {
```

Comment:
I think this enum should have documentation as well.

Reply:
fixed [866122b](https://github.com/salemove/ios-sdk-widgets/pull/778/commits/866122b5b8943a8a5435a943d4ffd351736af208)